docs/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="description" content="Tutorials & API docs for PAPB Test Frontend Lib">
     <link rel="icon" type="image/png" href="favicon.png">
     <link rel="stylesheet" type="text/css" href="index.css">
-    <title>PAPB Test Frontend Lib</title>
+    <title>papb-test-frontend-liest-frontend-lib</title>
```

Comment:
Looks like you made a mistake here.

```suggestion
    <title>papb-test-frontend-lib</title>
```